recipe/build-r-base.sh

```diff
@@ -457,8 +457,11 @@ Darwin() {
 
     if [[ "$target_platform" == "osx-arm64" ]]; then
       # For backwards compatibility
-      ln -sf ${PREFIX}/lib/libblas.dylib ${PREFIX}/lib/R/lib/libRblas.dylib
-      ln -sf ${PREFIX}/lib/liblapack.dylib ${PREFIX}/lib/R/lib/libRlapack.dylib
+      for lib in libblas.3 liblapack.3; do
+        # Use explicit relative path to prevent conda-build from using realpath
+        ln -sfn "../../${lib}.dylib" "${PREFIX}/lib/R/lib/${lib%.*}.dylib"
+        test -f "${PREFIX}/lib/R/lib/${lib}.dylib"
+      done
     fi
```

Comment:
We can remove this whole block for main branch.

Reply:
k, even better

Reply:
Ah, fuck. I pushed this to `main` instead of this branch.
Do you have admin rights on Azure to stop https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=965640&view=results from building?

Reply:
Done

Reply:
Thanks!